src/api/modules/utils.ts

```diff
@@ -135,8 +135,8 @@ export const appIdFromRegistry = (app: string, majorLocator: string) => {
 }
 
 const FREE_BILLING_OPTIONS_TYPE = 'free'
-export const isFreeApp = ({ type, free }: { type?: string; free?: boolean }) =>
-  type === FREE_BILLING_OPTIONS_TYPE || free
+export const isFreeApp = ({ type, free, plans }: { type?: string; free?: boolean; plans?: Plan[] }) =>
+  type === FREE_BILLING_OPTIONS_TYPE || free || !plans || plans.length === 0
```

Comment:
```suggestion
  type === FREE_BILLING_OPTIONS_TYPE || SPONSORED_BILLING_OPTION_TYPE || free
```

Reply:
Please do the appropriate changes on constants and CHANGELOG as well.

Reply:
So it isn't (and it will never be) possible for an app of type `sponsored` to be not free?

Reply:
It is a good question. For that I will summon @marlonguimaraes and @victorges to help.

Reply:
I don't think it makes sense to make a sponsored paid app, if this exists I'm not aware. 

Fixing this inside `isFreeApp` doesn't sound semantic to mee. Maybe it should be in the place where `isFreeApp` evaluated, like:

```js
if(isFreeApp() || isSponsoredApp()) {
...
}
```

Reply:
Agreed

Reply:
Agreed

Reply:
Well, if there will indeed never be a paid sponsored app, I totally agree. But, however strange that may sound, are we absolutely certain that it will never happen? Do we really intend to avoid the existence of a paid sponsored app?

Reply:
The reason is that the same problem that we have with a `sponsored` app could happen with a `billable` app, _i.e._, we could have a `billable` app that doesn't declare billing plans.

Reply:
> so that any app that doesn't declare billing plans is considered free

Are we sure that this won't change in the future as well?

I think it is safer to test for equality of `type` instead of testing for absence of `plans`.

Reply:
Ok, I will close the PR, as there is no agreement on the value of its core purpose. I'll open a new PR to fix the bug the users have reported.

Reply:
So, we have an contract in the `billingOptions` API. It's up to the `billingOptions.type` inform the type of the app, so VTEX must ensure that contract holds and consumers to use the API accordingly. 

If for whatever reason anywhere we are not making that contract work, it's a bug and we must fix the root cause.

If the consumers/users are not following the API, we can send them the appropriate documentation or create/improve the documentation if it's not existent, provide support, improve our documentation etc . (Example: https://vtex.io/docs/recipes/development/setting-your-apps-billing-model/)

(IMHO)